### DISCUSSION TITLE
When will this project be completed?

### DISCUSSION THREAD

**Discussion by coolxv** [JhnW/devana#2] [Q&A] [answered]

**Comment by JhnW** [comment, marked as answer]
Hi coolxv.
Please use https://github.com/JhnW/devana/discussions next time for questions like that.
First, we need to define "completed". Finished version (but still open to development and supplementing features) is a version with a working code generation module (the parser is basically finished). 
Hopefully **no later** than the end of the year we will enter the beta phase with the code generator working.
